clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/evaluation/TriggerAdapter.kt

```diff
@@ -119,6 +121,10 @@ class TriggerAdapter(triggerJSON: JSONObject) {
     val geoRadiusCount: Int
         get() = geoRadiusArray?.length() ?: 0
 
+    companion object {
+        const val KEY_FIRST_TIME_ONLY = "firstTimeOnly"
```

Comment:
This should probably be private (or the whole companion object)

Reply:
marking it as `internal` because it's used by unit tests. Fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/ec206d7a9d2ee6a018ac48d37bb4506471177f14